api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/core/Dense.kt

```diff
@@ -117,19 +119,22 @@ public class Dense(
         set(value) = assignWeights(value)
 
     private fun extractDenseWeights(): Map<String, Array<*>> {
```

Comment:
I checked other layers with bias parameter (convolution ones), and looks like they have similar problems:
- `weights` getter will try to extract bias parameters even though it is not enabled for the layer
- `paramCount` includes `bias` even if `useBias=false` has been used

Reply:
Yes, this is a typical pattern for all layers with the useBias flag. It looks like it requires common refactoring.
Could you please do it in this PR? It will be very helpful if you do it today or tomorrow. Do you have any time for that?

Reply:
Let's keep this idea for the future, it involves a lot of refactoring in the master branch which should become a release branch in two days. I suggest to finish this ticket without proposed DTO object